### PR TITLE
feat: add Stripe setttings base layout

### LINKED ISSUE #1706: [settings - Stripe] React page layout port
## Description

Once we have a React wrapper ( https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1680 ) and some testing capabilities ( https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1694 ), we can start adding some layout to the Stripe settings page.

The layout consists of:
- Wrapper around the components
- 2 tabs
- Settings sections
- Save button

Screen Shot 2021-08-19 at 10 48 55 AM

The 2 tabs and their behavior is handled in a separate ticket ( https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1707 )

## Acceptance criteria
- Layout is rendered
- A placeholder settings section is rendered as an example
- The save button is rendered

### PR DESCRIPTION
**Pull request by frosso**
## Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1706

Adds some base layout to the settings page.
I included a few packages:
- Emotion (used also by WP components)
- WP components
- WP i18n

Most of the styles/names come from WCPay.

## Testing instructions

- Ensure you have the `_wcstripe_feature_upe_settings` option in `wp_options` set to `'1'`
- Ensure your dependencies are up to date
- Run `npm run start`
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- A new layout should appear

Before:
Screen Shot 2021-08-19 at 12 09 40 PM

After:
Screen Shot 2021-08-20 at 8 49 33 AM

### CODE REVIEW

File: client/settings/card-body.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+import { CardBody } from '@wordpress/components';
+
+export default styled( CardBody )`
+	// increasing the specificity of the styles to override the Gutenberg ones
```
Comment (frosso) [inline client/settings/card-body.js:5]: These styles [come from WCPay](https://github.com/Automattic/woocommerce-payments/blob/f5e230660497ce4817dda56904b5a6451d8acf83/client/settings/card-body/styles.scss) - they should also apply here.

File: client/settings/settings-section/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+
+const SettingsSectionWrapper = styled.div`
```
Comment (frosso) [inline client/settings/settings-section/index.js:7]: These styles [also come from WCPay](https://github.com/Automattic/woocommerce-payments/blob/f5e230660497ce4817dda56904b5a6451d8acf83/client/settings/settings-section.scss)

**Comment by tommyshellberg** [comment]
Updating my PR for #1707 to use this as the base.

**Comment by harriswong** [comment]
👀

**Review by harriswong — APPROVED** [review]
Tested the branch and confirmed new layout with feature flag enabled. 👍

**Pull request merged**